controller/src/main/java/io/pravega/controller/store/stream/PersistentStreamBase.java

```diff
@@ -550,10 +560,11 @@ private long findSegmentSplitsMerges(List<StreamSegmentRecord> referenceSegments
                 }).thenApply(x -> segments);
     }
 
-    private CompletableFuture<Long> sizeBetweenStreamCuts(Map<Long, Long> streamCutFrom, Map<Long, Long> streamCutTo,
+    @VisibleForTesting
+    CompletableFuture<Long> sizeBetweenStreamCuts(Map<Long, Long> streamCutFrom, Map<Long, Long> streamCutTo,
```

Comment:
The description of the PR talks about:

> We now handle DataNotFound and return emptyMap for it (`sizeBetweenStreamCut`).

I'm not seeing it.

Reply:
The changes are to the method sizeBetweenStreamcut calls --> getSealedSegmentSizeMapShard
```
    CompletableFuture<SealedSegmentsMapShard> getSealedSegmentSizeMapShard(int shard) {
        return getSealedSegmentSizesMapShardData(shard)
                .handle((r, e) -> {
                    if (e != null) {
                        if (Exceptions.unwrap(e) instanceof DataNotFoundException) {
                            return SealedSegmentsMapShard.builder().shardNumber(shard).sealedSegmentsSizeMap(Collections.emptyMap()).build();
                        } 
                        throw new CompletionException(e);
                    } else {
                        return SealedSegmentsMapShard.fromBytes(r.getData());
                    }
                });
    }
```